src/migrations/20171015155157_reports_designated.ts

```diff
@@ -0,0 +1,26 @@
+import * as Knex from "knex";
+
+exports.up = async (knex: Knex): Promise<any> => {
+  return knex.schema.dropTableIfExists("reports_designated").then( (): void  => {
+    knex.schema.raw(`CREATE TABLE reports_designated (
+      "marketID" varchar(66) NOT NULL,
+      payout0 numeric,
+      payout1 numeric,
+      payout2 numeric,
+      payout3 numeric,
+      payout4 numeric,
+      payout5 numeric,
+      payout6 numeric,
+      payout7 numeric,
+      "isInvalid" integer
+    )`).then( (): void => {
+      knex.schema.table("reports_designated", (table: Knex.AlterTableBuilder): void => {
```

Comment:
This method for adding an autoincrement primary key doesn't work, even though I cribbed it from elsewhere in the project (reports); ideas why?

Reply:
You just found a bug. SQLite can't add a PRIMARY KEY column after the table has been made.

Irritating. I'll massage this into working.

Reply:
Blargh.

Hmmm. i wonder the best way to get it to work properly with pgsql and sqlite then, since pgsql does it significantly differently.

Reply:
BTW i did this in a different migration:

https://github.com/AugurProject/augur-node/pull/17/files#diff-0e33934ad8e27a733628d374a0e880e0R7

Reply:
So i've migrated these tables to the createTable() format, but i don't see Knex supporting a "numeric" data type. We could go straight to decimal, which is more explicitly what we want, but then is that compatible/reasonable with pg?